### PR TITLE
Graphics page

### PR DESCRIPTION
**Pull request by johnkuney**
Creating graphics page for logos and visual identity.

Given these items do not change often and when they do likely all change at once, and that the downloadable images are not shown on the page, I opted to put the images in the static folder as it is much easier to manage than importing like we're doing with exchanges etc.

### CODE REVIEW

File: v2/src/pages/graphics.js [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+              />
+            </div>
+          </Col>
+          <Col md={6} className={S.colorctnpadding}>
+            <Row>
+              <Col xs={4} className={S.colorctn}>
+                <span
+                  className={S.boxcolor}
+                  style={{ background: "#F59332" }}
+                ></span>
```
Comment (deadalnix) [inline v2/src/pages/graphics.js:295]: You can use a self closing tag

**Review by deadalnix — APPROVED** [review]

**Review by deadalnix — CHANGES_REQUESTED** [review]
You need to update the links.

File: v2/src/pages/graphics.js (hunk 1 above)
Comment (johnkuney) [reply v2/src/pages/graphics.js]: Ok sure updated

**Comment by johnkuney** [comment]
Okay link in menu has been updated. The only reason I left it off is because wont it cause merge conflicts if I change the header links for different PRs? Thats why I only did one of them and was planning on doing the other right after...

File: v2/src/pages/graphics.js [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+import AcceptedHere from "assets/images/logos/bitcoin-cash-accepted-here.jpg"
+import AcceptedHereOrange from "assets/images/logos/bitcoin-cash-logo-accepted-here-orange.jpg"
+import Donations from "assets/images/logos/bitcoin-cash-logo-donations-accepted.jpg"
+import DonationsOrange from "assets/images/logos/bitcoin-cash-logo-donations-accepted-orange.jpg"
+import DownloadIcon from "assets/icons/download.svg"
+
+const GraphicsPage = () => {
+  return (
+    <>
+      <SEO title="Bitcoin Cash Wallets" />
```
Comment (deadalnix) [inline v2/src/pages/graphics.js:24]: ???

Comment (johnkuney) [reply v2/src/pages/graphics.js:24]: ahh sorry missed it. Will update

**Pull request merged**